app/Actions/Desktop/SetupDesktopAdmin.php

```
@@ -0,0 +1,35 @@
+        return DB::transaction(function () use ($input): User {
+            $user = app(CreateUser::class)->create([
+                'name' => $input['name'] ?? null,
+                'email' => $input['email'] ?? null,
+                'password' => $input['password'] ?? null,
+                'role' => UserRole::ADMIN->value,
+            ]);
+
+            $user->ensureHasDefaultProject();
+
+            return $user;
+        });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'create'. (undefined)

(StaticAccess)